src/api-service/__app__/onefuzzlib/azure/containers.py

```diff
@@ -6,76 +6,108 @@
 import datetime
 import os
 import urllib.parse
-from typing import Dict, Optional, Union, cast
+from enum import Enum
+from typing import Any, Dict, Optional, Union, cast
 
 from azure.common import AzureHttpError, AzureMissingResourceHttpError
 from azure.storage.blob import BlobPermissions, ContainerPermissions
 from memoization import cached
 
-from .creds import get_blob_service
+from .creds import get_blob_service, get_func_storage, get_fuzz_storage
+
+
+class StorageType(Enum):
+    corpus = "corpus"
+    config = "config"
+
+
+def get_account_id_by_type(storage_type: StorageType) -> str:
+    if storage_type == StorageType.corpus:
+        account_id = get_fuzz_storage()
```

Comment:
Is `corpus` -> `fuzz..` and `config` -> `func..` a relationship we expect to always hold? If so, we should consider explicitly documenting it.

Reply:
As I continue to address the issues relating to storage account sharding, I expect this to change, which is why I have not described this in developer-focused documentation yet.